package/appmarkable/package

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2020 The Toltec Contributors
 # SPDX-License-Identifier: MIT
 
-pkgname=appmarkable
+pkgnames=(appmarkable)
```

Comment:
is it easy to support both pkgname and pkgnames?

Reply:
It should be easy enough yes. When considering the two options, the possible confusion caused by having two similarly-named fields in the `docs/package.md` document came to mind. IMO it’s less confusing if we only have `pkgnames`.